ui/src/plugin/PluginDetailView.tsx

```diff
@@ -0,0 +1,272 @@
+                                await this.refreshFeatures();
+                            }}
+                        />
+                    </PanelWrapper>
+                ) : null}{' '}
+                {capabilities.indexOf('displayer') !== -1 ? (
+                    <PanelWrapper
+                        name={'Displayer'}
+                        description={'This is the information generated by the plugin.'}
+                        refresh={this.refreshDisplayer.bind(this)}
+                        icon={Subject}>
+                        <DisplayerPanel
+                            pluginInfo={pluginInfo}
+                            displayText={this.state.displayText || 'Loading...'}
+                        />
+                    </PanelWrapper>
+                ) : null}
+            </DefaultPage>
+        );
+    }
+}
+
+interface IPanelWrapperProps {
+    name: string;
+    description?: string;
+    refresh?: () => Promise<void>;
+    icon?: React.ComponentType;
+}
+
+const PanelWrapper: React.SFC<IPanelWrapperProps> = ({
+    name,
+    description,
+    refresh,
+    icon,
+    children,
+}) => {
+    const Icon = icon;
+    return (
+        <Container style={{display: 'block', width: '100%', margin: '20px', color: 'white'}}>
+            <Typography variant="headline">
+                {Icon ? (
+                    <span>
+                        <Icon />
+                        &nbsp;
+                    </span>
+                ) : null}
+                {name}
+                {refresh ? (
+                    <Button
+                        style={{float: 'right'}}
+                        onClick={() => {
+                            refresh();
+                        }}>
+                        <Refresh />
+                    </Button>
+                ) : null}
+            </Typography>
+            {description ? (
+                <Typography variant="subheading" style={{color: 'grey'}}>
+                    {description}
+                </Typography>
+            ) : null}
+            <hr />
+            <div
+                className={name
+                    .toLowerCase()
+                    .trim()
+                    .replace(/ /g, '-')}>
+                {children}
+            </div>
+        </Container>
+    );
+};
+
+interface IConfigurerPanelProps {
+    pluginInfo: IPlugin;
+    initialConfig: string;
+    save: (newConfig: string) => Promise<void>;
+}
+class ConfigurerPanel extends Component<IConfigurerPanelProps, {unsavedChanges: string}> {
+    public state = {unsavedChanges: ''};
+
+    public render() {
+        return (
+            <div>
+                <MonacoEditor
+                    language="yaml"
+                    theme="vs-dark"
+                    height="15em"
+                    ref="monaco"
+                    value={
+                        this.state.unsavedChanges
```

Comment:
The unsaved changes may be empty (when deleting all content) so maybe set it to false and explicitly check for it. Otherwise it is not possible to `ctrl-a backspace`.